actions/code-release/action.yml

```diff
@@ -5,47 +5,32 @@ inputs:
    token:
      description: "Token needed to access files "
      required: true
-   path-to-json:
-     description: "Path to json folder with version"
+   access-key:
+     description: "AWS access key"
      required: true
-   repository-name:
-     description: "Name of repository needed to create code packages"
+   secret-key:
+     description: "AWS secret key"
      required: true
-   repository-type:
-     description: "Repository type used in bash code"
+   json-path:
+     description: "path to json with version number"
      required: true
-   staging-bucket:
-     description: "Staging s3 bucket name"
+   lambda-name:
+     description: "lambda name"
      required: true
-   production-bucket:
-     description: "Production s3 bucket name"
+   s3-bucket:
+     description: "s3 bucket name"
      required: true
-   access-key-staging:
-     description: "AWS staging key"
+   txt-file-name:
+     description: "Name of the created txt file"
      required: true
-   secret-key-staging:
-     description: "AWS secret key staging"
+   working-folder-name:
+     description: "Name of the folder in which the zip and txt files were created"
      required: true
-   access-key-production:
-     description: "AWS production key"
+   ARN-number:
```

Comment:
This is not ARN number. IT is AWS account number

Reply:
name changed